www/include/Administration/configChangelog/viewLogs.php

```diff
@@ -52,7 +52,7 @@ function searchUserName($user_name)
         if ($str != "") {
             $str .= ", ";
         }
-        $str .= "'" . $row['contact_id'] . "'";
+        $str .= "'" . $pearDB->escape($row['contact_id']) . "'";
```

Comment:
Why do you want to escape an integer value ?
In the centreon database, the structure of 'contact' indicates that contact_id is an integer.
When you are supposed to work with an integer value, please use the following syntax `(int) $row['contact_id']`.